ui/main/src/app/services/templateGateway/UserCardTemplateGateway.ts

```
@@ -34,6 +34,8 @@ export class UserCardTemplateGateway {
     private static _functionToGetSpecficCardInformationFromTemplate: Function;
     private static _functionToSendEntityUsedForSendingCardToTemplate: Function;
 
+    private static _functionToBeCalledBeforeCardSending: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)